user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -152,28 +128,35 @@ The elements will be created with the following specifications:
 
 1. Below the **Create Bulk** script text, enter the path of the CSV file.
 
+   Here is an example of a CSV file with the mandatory headers and the content of each row:
+
+   | ElementName | Ccap_DMA | Collector_DMA | Protocol | IpAddress | GetCommunityString | SetCommunityString | GetCommunityStringColl | SetCommunityStringColl | Network | Market | Hub | SystemUser | SystemPass |
+   |---|---|---|---|---|---|---|---|---|---|---|---|---|---|
+   | filename1 | EPM-DMA03 | EPM-DMA03 | CISCO CBR-8 CCAP Platform | 10.11.12.11 | getPublic | setprivate | collectorget | collectorset | GLOBAL NETWORK | EAST MARKET 01 | EAST HUB 01 | US1 | 123 |
+   | filename2 | EPM-DMA04 | EPM-DMA04 | CISCO CBR-8 CCAP Platform | 10.11.12.12 | getprivate | setPublic | collectorget | collectorset | GLOBAL NETWORK | EAST MARKET 01 | EAST HUB 01 | US2 | 123 |
+
    > [!NOTE]
    >
    > - The CSV file must be present on the server running the DMA.
-   > - The CSV file has to have the following structure: ElementName, Ccap_DMA, Collector_DMA, Protocol, IpAddress, GetCommunityString, SetCommunityString,GetCommunityStringColl, SetCommunityStringColl, Network, Market, Hub, SystemUser, SystemPass.
+   > - The CSV file has to have the following structure: ElementName*, Ccap_DMA*, Collector_DMA*, Protocol*, IpAddress*, GetCommunityString, SetCommunityString, GetCommunityStringColl, SetCommunityStringColl, Network, Market, Hub, SystemUser, SystemPass.
+   > - For the **Protocol** column, only the following values are currently accepted:
+   >   - CISCO CMTS CCAP Platform
+   >   - Arris E6000 CCAP Platform
+   >   - Casa Systems CCAP Platform
+   >   - CISCO CBR-8 CCAP Platform
+   >   - Huawei 5688-5800 CCAP Platform
 
 1. Click *Create Bulk*.
 
-The elements will be created with the specifications in the CSV file of each row.
+   The elements will be created with the specifications in the CSV file of each row.
 
-Here is an example of the CSV file with the mandatory headers and the content of each row.
+   > [!NOTE]
+   >
+   > - Each time the script is executed, it will attempt to create five elements. If there are fewer than five elements left to create, it will create as many as possible without any issues. After the first set of elements has been created, you can execute the script again without modifying the CSV file, and it will attempt to create the next set of five elements.
+   > - Click the *Cancel* button in any window to close the Automation script at any time.
```

Comment:
Maybe we can include *not as a note* that the script will provision devices in bulk of 5 elements.

Reply:
Information included not as a note